docs/source/reference/howto/wipe_persistent_data.rst

```diff
@@ -56,41 +72,37 @@ in the sequence given below, so you shouldn't proceed to next steps until the pr
 
       $ cchq <env_name> ap wipe_kafka.yml
 
+#. Remove the "wipe_environment_enabled: True" line in your `public.yml` file.
```

Comment:
is this a new option that has been added? "wipe_environment_enabled"?

Reply:
No it is used in ansible playbooks that wipe data as an extra safety feature to prevent people from accidentally running those playbooks. I just moved it to "wrap" all of the steps in the wiping data section even though it doesn't apply to django management commands because I thought that made more logical sense.

Reply:
This option was added when we added the Ansible tasks for nuking environments' data, to ensure that the people with their fingers on the button know what they're doing.

![cat-north-korea](https://github.com/dimagi/commcare-cloud/assets/708421/53355e35-e8a0-4526-9719-6c88a6d0dbb6)